expression/builtin_time.go

```diff
@@ -5619,7 +5619,8 @@ func (b *builtinLastDaySig) evalTime(row chunk.Row) (types.Time, bool, error) {
 		return types.Time{}, true, handleInvalidTimeError(b.ctx, err)
 	}
 	tm := arg.Time
-	year, month, day := tm.Year(), tm.Month(), 30
+	var day int
```

Comment:
The value of `day` is 30 in the original code, why change it?

Reply:
Next time , it is used in `day = types.GetLastDay(year, month)`
It has been already overwritten.
I think `30` is never used.